assets/azure.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: managed-premium-retain
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: kubernetes.io/azure-disk
+reclaimPolicy: Retain
```

Comment:
I think it should be Delete, i.e. unset.

Reply:
this is only to be consistent with other clouds. Whether admin wants their disks automatically cleaned up is a matter of preference/policy I think, so staying consistent would be better IMO.

Reply:
removed